lesson-6/twenty-one.js

```diff
@@ -66,6 +74,12 @@ function displayHand(hand) {
   return listOfCards;
 }
 
+function displayHandAndTotal(playerHand, dealerHand, playerPoints) {
```

Comment:
new function created to display hands for player and dealer, as well as player points (was originally in body of code, extracted to its own function)